doc/DOCUMENTATION.org

```diff
@@ -550,6 +551,21 @@ change the location of this directory.
 so =~/.spacemacs= must not exist for =~/.spacemacs.d/init.el= to be used by
 Spacemacs.
 
+*** Chemacs users with =$CONFIG_HOME/.spacemacs.d= as =$SPACEMACSDIR=
```

Comment:
CONFIG_HOME may not be set on some systems. how do you deal with that?

Reply:
Woops! That's just a bad referent; I'm not using CONFIG_HOME in the code. I should have referred to HOME.

The purpose of the heading was to guide users who are using `spacemacs.d` as their `.emacs.d`; `CONFIG_HOME` was unnecessary.

I'll improve the clarity of this heading.